security/pkg/nodeagent/sds/server.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sds
+
+import (
+	"net"
+
+	"google.golang.org/grpc"
+
+	"istio.io/istio/pkg/log"
+)
+
+const maxStreams = 100000
+
+// Args provides all of the configuration parameters for secret discovery service.
+type Args struct {
```

Comment:
For constructor parameters, we use namely "Options" mostly.

Reply:
personally I don't have preference on this, just follow naming convention from pilot server(https://github.com/istio/istio/blob/master/pilot/pkg/bootstrap/server.go#L107).

Reply:
I still want us to be consistent with the rest of security code naming.

If you search `New.*Option f:thirdy_party` for golang standarad libraies in internal code search website, you can see way more occurrences than `New.*Args f:thirdy_party`.

Reply:
done